server.r

```diff
@@ -1204,7 +1204,8 @@ shinyServer(function(input, output, session){
             # Save summary table
             echo("")
             echo("Please save the summary table using the popup window.")
-            path <- tclvalue(tkgetSaveFile(initialfile = paste0("TreatmentSummary--", input$SummaryMeasurements, ".csv")))
+            all_measurements <- paste(input$SummaryMeasurements, collapse = "--")
```

Comment:
@hmbotelho I do not get the logic here.
I assume `input$SummaryMeasurements` can have multiple entries, right?
Since you collapse them later, does that mean we save one excel file for all the measurements?

Reply:
Yes, `input$SummaryMeasurements` is where you select the measurements you want to summarise.
There you can select one or more entries.

The change above is to avoid having funny default file names when summarizing more than 1 measurement.
*e.g.*: `TreatmentSummary--measurement1.csv TreatmentSummary--measurement2.csv TreatmentSummary--measurement3.csv`
is converted into
`"TreatmentSummary--measurement1--measurement2--measurement3.csv"`

The output is always 1 csv file, regardless of the number of measurements. Would you prefer otherwise?

Reply:
No, that's fine, but how does it look? Are the column names then indicating which measurement it is?
Or do you concatenate row wise?
Could you maybe mail me an example output?

Reply:
Tables are concatenated row wise. The first column indicates which measurement it refers to.

* [Summary for 1 measurement](https://github.com/hmbotelho/shinyHTM-1/blob/master/tests/normalization/test1_output.csv)
* [Summary for 3 measurements](https://github.com/hmbotelho/shinyHTM-1/blob/master/tests/normalization/test2_output.csv)